docs/pages/docs/components/_slug.vue

```diff
@@ -11,7 +11,7 @@
         :key="meta.component"
         :component="meta.component"
         :events="meta.events"
-        :root-event-emitters="meta.rootEventEmitters"
+        :root-event-emitters="meta.rootEventListeners"
```

Comment:
Need to change the prop ` :root-event-emitters="meta.rootEventListeners"` to ` :root-event-listeners="meta.rootEventListeners"`